app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,18 @@
+class CommentsController < ApplicationController
```

Comment:
- [ ] _New_ action for _comments_ is missing. Kindly ensure to define _new_ action for _comments_ controller, similar to:
```ruby
   def new
     @comment = Comment.new
   end
```

Reply:
@rloterh ,

Thank you for the feedback. You requested changes and you didn't give me any reason why the changes are required. Is the app not working as expected or do you get any error when you try to run migrate?

Reply:
Hi @micheaol 

It's a nice idea to implement partials in your project. However, this project requires that you create a pair of _routes_ in the `config/routes.rb` file for creating _comments_. Hence the need for _new_ action in _comments_controller.

Reply:
@rloterh ,
Thank you for the feedback. But you never answered any of my questions:

1. Did you get any error messages when you run the app?
2. Is the app not working as expected?